js/components/SpidLoginButton.js

```diff
@@ -237,6 +220,21 @@ class IdpSelectionScreen extends React.Component {
     })
   }
 
+  createDemoButton() {
+    return (
+      <Button iconRight light block key={demoIdp.id} style={StyleSheet.flatten(styles.idpButton)} onPress={() => {
```

Comment:
perhaps you can create a shared `createButton()` method with the button UI and call it from `createDemoButton` and from `createButtons` to avoid duplicating the UI code